python/cudf/cudf/tests/test_index.py

```diff
@@ -2886,3 +2886,20 @@ def test_index_to_pandas_nullable(data, expected_dtype):
     expected = pd.Index(data, dtype=expected_dtype)
 
     assert_eq(pi, expected)
+
+
+class TestIndexScalarGetItem:
+    @pytest.fixture(
+        params=[range(1, 10, 2), [1, 2, 3], ["a", "b", "c"], [1.5, 2.5, 3.5]]
+    )
+    def index_values(self, request):
+        return request.param
+
+    @pytest.fixture(params=[int, np.int8, np.int32, np.int64])
+    def i(self, request):
+        return request.param(1)
+
+    def test_scalar_getitem(self, index_values, i):
+        index = cudf.Index(index_values)
+
+        assert not isinstance(index[i], cudf.Index)
```

Comment:
```suggestion
        g_index = cudf.Index(index_values)
        p_index = g_index.to_pandas()

        assert not isinstance(g_index[i], cudf.Index)
        assert_eq(p_index[i], g_index[i])
```
Can we also assert the returned values?

Reply:
Done.